model/file_info.go

```diff
@@ -189,6 +190,10 @@ func GetEtagForFileInfos(infos []*FileInfo) string {
 	return Etag(infos[0].PostId, maxUpdateAt)
 }
 
+func (fi *FileInfo) CanBeUsedForVoiceMessage() bool {
+	return strings.Contains(VoiceMessagesSupportedMimeTypes, strings.ToLower(fi.MimeType))
```

Comment:
I wonder if we should simply use [`EqualFold`](https://pkg.go.dev/strings#EqualFold)?

Reply:
`VoiceMessagesSupportedMimeTypes` is a comma-separated string list so want can't rely on "equality".